docs/azmcp-commands.md

```diff
@@ -18,18 +19,24 @@ The following args are available for all commands:
 ## Available Commands
 
 ### Server Operations
+---
```

Comment:
Are these needed?

Reply:
They get added by VSCode automatically.  I'll remove them now to cleanup the PR